design-tokens/tokens/layer1/base.json

```diff
@@ -333,7 +387,7 @@
             "value": {
               "fontFamily": "{echoes.fontFamily.sans}",
               "fontWeight": "{echoes.fontWeight.regular}",
-              "lineHeight": "{echoes.lineHeight.body}",
+              "lineHeight": "{echoes.lineHeight.lineHeight.body}",
```

Comment:
Let's reference the new lineHeight values no? Same for all the following lineHeights.

And same for the fontsizes.